nltk/translate/api.py

```diff
@@ -46,7 +47,7 @@ class AlignedSent(object):
     :type alignment: Alignment
     """
 
-    def __init__(self, words=[], mots=[], alignment='', encoding='utf8'):
+    def __init__(self, words=[], mots=[], alignment=[], encoding='utf8'):
```

Comment:
@kmike – that's true – I don't see any use for these defaults so I'll remove them.
@alvations – I'll drop `encoding` too, thanks.